outlines/prompts.py

```diff
@@ -166,11 +199,22 @@ def prompt(fn: Callable) -> Prompt:
     if docstring is None:
         raise TypeError("Could not find a template in the function's docstring.")
 
-    template = Prompt._template_from_str(cast(str, docstring))
+    template = Prompt._template_from_str(cast(str, docstring), filters)
 
     return Prompt(template, signature)
 
 
+def _add_filters(
+    env: jinja2.Environment, filters: Union[List[Callable], Dict[str, Callable]]
+):
+    if isinstance(filters, list):
+        for filter_fn in filters:
+            env.filters[filter_fn.__name__] = filter_fn
```

Comment:
Let's stick to `filters` being a dictionary.

Reply:
I removed the list option.